Sources/DbPackages-mysql.php

```diff
@@ -433,7 +433,7 @@ function smf_db_change_column($table_name, $old_column, $column_info)
 	$smcFunc['db_query']('', '
 		ALTER TABLE ' . $table_name . '
 		CHANGE COLUMN `' . $old_column . '` `' . $column_info['name'] . '` ' . $type . ' ' . (!empty($unsigned) ? $unsigned : '') . (!empty($column_info['not_null']) ? 'NOT NULL' : '') . ' ' .
-				(empty($default) ? '' : 'default ' . $default) . ' ' .
+				($default === '' ? '' : 'default ' . $default) . ' ' .
```

Comment:
this goesa now in direction of nit-picking so feel free to ignore,
but i would like to see that this ``$default === ''`` operator is the like in pg,
so choose what you want === or !== but use this one in both files.